addons/web/static/tests/views/view_dialogs_tests.js

```diff
@@ -604,7 +604,9 @@ QUnit.module('Views', {
         new dialogs.SelectCreateDialog(parent, {
             res_model: 'partner',
             on_selected: function(records) {
-                assert.equal(records.length, 3)
+                assert.equal(records.length, 3);
+                assert.ok(records.every(e => 'display_name' in e));
+                assert.ok(records.every(e => 'id' in e));
```

Comment:
`assert.strictEqual(records.map((r) => r.display_name).toString(), "machin,truc,bidule");`

Same for id. That way, you check the values are correct (the key might there, but value could be `undefined`)